_posts/kubernetes/1200-09-01-K8-general-kubernetes-questions.md

```diff
@@ -35,9 +35,41 @@ It is possible to deploy an ingress resource in the same way you can deploy any
 ## Can V1 kube and V2 kube live on the same cluster?
 Yes. However, in the Spinnaker UI you will see every K8s resource twice. Once, when the V1 provider sees it, and once, when the V2 provider sees it.
 
+## is it possible to apply secrets using https://github.com/futuresimple/helm-secrets?
+Today that is not possible
 
+## How do we handle deployment in multiples cluster? For example, if our environment includes more than one k8s cluster
+With the account concept in K8s, an account is just a k8s cluster, when you configure an account you configure it to use an specific kube config file, this define how we communicate and authenticate on our cluster, if you have multiple cluster you can configure multiple accounts within spinnaker using different kube config files that have the address for the API, when you set the deployment step you setup a different account to deploy to and spinnaker will use the credentials found in the kube config.
 
+## There are two different types of pipelines : Startegy and Pipeline , what's the difference.
+A pipeline is generally just the way your application rolls out, that would be like saying deploying to a particular environment manual judgment deploying into another, we do things like kicking of jenkins jobs. Strategy is kind of a way of taking all of the different pieces of that pipeline and being able to select them as a way you want to deploy. 
 
+## It seems that Spinnaker is polling data from Kubernetes. How often is this data refreshed?
+By default spinnaker tries to pull every 30 seconds, but you can configure this time.
 
+## How's the performance of Spinnaker when it's keeping track of hundreds of pipelines?
+Spinnaker was build to run at scale, hundreds of pipelines are safe for spinnaker but comes with the caviat that you have at some extend manage and scale spinnaker controlers. We have at spinnaker.io a little guide at what compponents to scale horizontall, depending on how your work looks like, in the case of pipelines there is a component called front50, you have to probably add more replicas of that so you don´t run into issues when running large number of pipelines
```

Comment:
This needs to be reworked a bit. Let's try:
```
Spinnaker was built to run at scale, hundreds of pipelines are safe with Spinnaker, but that comes with the caviat that you have to manage and scale Spinnaker components. We have a guide at [spinnaker.io](https://spinnaker.io) that describes which components can scale horizontally, depending on your workload. In the case of pipelines, there is a component called Front50 and you can add more replicas to prevent issues with large numbers of pipelines.
```

Reply:
check